CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-vue-barcode-scanner ChangeLog
 
+## 1.3.0 - 2025-03-xx
+
+### Changed
+- Add zoom icons to slider.
+- Prevent pinch to zoom.
```

Comment:
```suggestion
- Prevent pinch to zoom (since it presently scales the website and
  not the camera, which can be confusing UX). A future update
  might add pinch to zoom that scales the camera.
```